Sources/AuthFoundation/Requests/Token+Requests.swift

```diff
@@ -82,10 +82,12 @@ extension Token.IntrospectRequest: OAuth2APIRequest, APIRequestBody {
     var httpMethod: APIRequestMethod { .post }
     var contentType: APIContentType? { .formEncoded }
     var acceptsType: APIContentType? { .json }
-    var authorization: APIAuthorization? { token }
+    var authorization: APIAuthorization? { nil }
+    private var clientId: String? { token.idToken?[.clientId] }
```

Comment:
You can delete this line, since it's not used.

Reply:
I must've missed this, thanks. I have updated this.